Makefile

```
@@ -12,6 +12,8 @@ clean:
 
 test: build
 	bundle exec rspec spec
+	# must fix existing accessibility errors before uncommenting e2e tests
+	# node --test spec/e2e/
```

Related to previous comment:

```suggestion
	node --test spec/e2e/
```